src/Client.php

```diff
@@ -196,6 +198,17 @@ public function removeCachedMultiCurrency()
         unlink(self::MULTI_CURRENCY_CACHE_FILE_PATH);
     }
 
+    public function getIdealIssuers(): IdealIssuers
+    {
+        $response = new IdealIssuers();
+        foreach ($this->api_client->getIdealIssuers() as $issuer) {
```

Comment:
In this class `Client.php` all functions except createClient(), getApiClient() and getIdealIssuers() has descriptions like this:
 /**
     * @return $response
     */

Is such a description of functions important?

Reply:
I prefer to use one code style for all functions, with description is always better, basically because IDE use it to provide more information in runtime coding. Will add description